src/Explorer/Explorer.tsx

```diff
@@ -875,9 +924,26 @@
       handleError(error, "Explorer/onNewNotebookClicked");
       throw new Error(error);
     }
+    if (userContext.features.notebooksTemporarilyDown === false && userContext.features.phoenix === true) {
+      useDialog.getState().showOkCancelModalDialog(
+        Notebook.newNotebookModalTitle,
+        Notebook.newNotebookModalContent,
+        "Create",
+        async () => {
+          await this.allocateContainer();
+          parent = parent || this.resourceTree.myNotebooksContentRoot;
+          this.createNewNoteBook(parent, isGithubTree);
+        },
+        "Cancel",
+        undefined
+      );
+    } else {
+      parent = parent || this.resourceTree.myNotebooksContentRoot;
+      this.createNewNoteBook(parent, isGithubTree);
```

Comment:
>isGithubTree [](http://example.com/codeflow?start=37&length=12)

wouldnt this be true since it is in urrlse part? do we need to chk for !this flag in if statemtn